src/wp-admin/includes/class-wp-automatic-updater.php

```diff
@@ -1828,6 +1838,52 @@ protected function has_fatal_error() {
 			$result       = json_decode( trim( $error_output ), true );
 		}
 
+		if ( defined( 'WP_DEBUG' ) && WP_DEBUG ) {
+			$fatal_error = null;
+
+			$last_error = error_get_last();
+			if ( $last_error && in_array( $last_error['type'], array( E_ERROR, E_PARSE, E_COMPILE_ERROR, E_USER_ERROR ) ) ) {
+				$fatal_error = "PHP Fatal error: {$last_error['message']} in {$last_error['file']} on line {$last_error['line']}";
```

Comment:
Should this be translated? I suppose it wouldn't be translated in the error log either.